### ISSUE TITLE
A11y issues in Tabs Angular 9

### ISSUE THREAD

**Issue by Shijir** [vmware/clarity#4332]
## Describe the bug

With changes from Angular 9, the keyboard focus navigation doesn't work properly in Tabs. 
Moreover, our Tabs component had other underlying a11y issues and most of them are involved around the tab overflow. Overall, the current issues are:

1. When we navigate through tab links using your arrow keys and when the tab overflow opens, it doesn't focus on the first link in the overflow content. The same issue is present when the mouse click is used. 
 
2. When the overflow closes while the focus is on the last item in it, and a user presses the left arrow key, it opens the closed overflow again. The expected behavior is it should focus the last visible tab link.

3. We implement manual tab activation as users could try to potentially load async content in the tab panel. So we need to make only the active tab link tabbable, which is not the case in our current implementation. 

## Versions

**App**

* Angular: v9
* Node: [e.g. 8.10.0]
* Clarity: v3

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]
